interface/forms/eye_mag/php/eye_mag_functions.php

```diff
@@ -4048,21 +4048,6 @@ function report_header($pid, $direction = 'shell')
     ?>
     <table style="width:100%;">
         <tr>
-            <td style='width:150px;text-align:top;'>
-                <?php
-                if ($direction == "web") {
-                    $practice_logo = $GLOBALS['webroot']."/sites/default/images/practice_logo.gif";
-                    if (file_exists($OE_SITE_DIR."/images/practice_logo.gif")) {
-                        echo "<img src='$practice_logo' align='left' style='width:150px;margin:0px 10px;'><br />\n";
-                    }
-                } else {
-                    $practice_logo = "$OE_SITE_DIR/images/practice_logo.gif";
-                    if (file_exists($practice_logo)) {
-                        echo "<img src='$practice_logo' align='left' style='width:100px;margin:0px 10px;'><br />\n";
-                    }
-                }
-            ?>
-            </td>
```

Comment:
This code needs to be added back

Reply:
I added it back :) That image doesn't exist, though. How should I fix this?